components/widgets/admin_console/admin_panel_with_link.tsx

```diff
@@ -6,14 +6,28 @@ import PropTypes from 'prop-types';
 import {FormattedMessage} from 'react-intl';
 import {Link} from 'react-router-dom';
 
-import AdminPanel from './admin_panel.jsx';
+import AdminPanel from './admin_panel';
 
-const AdminPanelWithLink = (props) => {
+type Props = {
+    children?: React.ReactNode,
+    className: string,
+    id?: string,
+    titleId: string,
+    titleDefault: string,
+    subtitleId: string,
+    subtitleDefault: string,
+    url: string,
+    disabled: boolean,
+    linkTextId: string,
+    linkTextDefault: string,
+}
+
+const AdminPanelWithLink = (props: Props) => {
     const button = (
         <Link
             className='btn btn-primary'
             to={props.url}
-            disabled={props.disabled}
+            onClick={props.disabled ? (e) => e.preventDefault() : () => null}
```

Comment:
won't this break styling of Link? (removal of disabled prop)

Reply:
It shouldn't, the `a` element doesn't have `disabled` attribute.